music21/scale/__init__.py

```diff
@@ -1358,15 +1355,14 @@ def __eq__(self, other):
         if not self.isConcrete or not other.isConcrete:
             # if tonic is none, then we automatically do an abstract comparison
             return self._abstract == other._abstract
-        else:
-            if (isinstance(other, self.__class__)
-                    and isinstance(self, other.__class__)
-                    and self._abstract == other._abstract
-                    and self.boundRange == other.boundRange
-                    and self.tonic == other.tonic):
-                return True
-            else:
-                return False
+
+        if (isinstance(other, self.__class__)
```

Comment:
as above.  Please stop changing code style that does not increase speed, just to match your code style.